### PR TITLE
benchmarkv3: refactoring the main logic

### PR DESCRIPTION
**Pull request by xiang90**
/cc @yichengq

### CODE REVIEW

File: tools/v3benchmark/main.go [hunk 1]
```diff
@@ -50,53 +55,32 @@ func main() {
...
-
-			for _ = range jobs {
-				st := time.Now()
-				resp, err := etcd.Range(context.Background(), req)
+	requests := make(chan struct{}, n)
+	conn, err := grpc.Dial(url)
+	if err != nil {
+		fmt.Errorf("dial error: %v", err)
+		os.Exit(1)
+	}
```
Comment (yichengq) [inline tools/v3benchmark/main.go]: I think either using one conn for all concurrency or using one conn for one concurrency cannot cover the real case. In the real life, there should be one conn from one client machine to the server.

File: tools/v3benchmark/get.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"time"
+
+	"github.com/coreos/etcd/etcdserver/etcdserverpb"
+	"golang.org/x/net/context"
```
Comment (yichengq) [inline tools/v3benchmark/get.go]: godeps

File: tools/v3benchmark/main.go (hunk 1 above)
Comment (xiang90) [inline tools/v3benchmark/main.go]: I will just stick to what gRPC official does today. I would like to explore more in the future.

Comment (yichengq) [inline tools/v3benchmark/main.go]: that's fair.

**Comment by yichengq** [comment]
lgtm after addressing godeps

**Pull request merged**